framework/src/Volo.Abp.UI/Localization/Resources/AbpUi/de.json

```diff
@@ -4,43 +4,50 @@
     "Languages": "Sprachen",
     "AreYouSure": "Sind Sie sicher?",
     "Cancel": "Abbrechen",
+    "Clear": "Bereinigen",
     "Yes": "Ja",
     "No": "Nein",
     "Ok": "Ok",
     "Close": "Schließen",
     "Save": "Speichern",
-    "SavingWithThreeDot": "Speichere...",
+    "SavingWithThreeDot": "Speichern...",
     "Actions": "Aktionen",
     "Delete": "Löschen",
     "SuccessfullyDeleted": "Erfolgreich gelöscht",
     "Edit": "Bearbeiten",
     "Refresh": "Aktualisieren",
     "Language": "Sprache",
     "LoadMore": "Mehr laden",
-    "ProcessingWithThreeDot": "Verarbeite...",
-    "LoadingWithThreeDot": "Lade...",
-    "Welcome": "Willkommen",
+    "ProcessingWithThreeDot": "Wird Bearbeitet...",
+    "LoadingWithThreeDot": "Wird geladen...",
+    "Welcome": "Wilkommen",
     "Login": "Anmelden",
     "Register": "Registrieren",
     "Logout": "Abmelden",
-    "Submit": "Absenden",
+    "Submit": "Senden",
     "Back": "Zurück",
-    "PagerSearch": "Suchen",
+    "PagerSearch": "Suche",
     "PagerNext": "Nächste",
     "PagerPrevious": "Vorherige",
     "PagerFirst": "Erste",
     "PagerLast": "Letzte",
-    "PagerInfo": "Zeige _START_ bis _END_ von _TOTAL_ Einträgen",
-    "PagerInfo{0}{1}{2}": "Zeige {0} bis {1} von {2} Einträgen",
-    "PagerInfoEmpty": "Zeige 0 bis 0 von 0 Einträgen",
-    "PagerInfoFiltered": "(gefiltert von _MAX_ Einträgen insgesamt)",
+    "PagerInfo": "Anzeigen von _START_ bis _END_ von _TOTAL_ Einträgen",
```

Comment:
This sounds like a "command". Since this is a information only, the previous wording suits better in my opinion.

Reply:
since it is paging information that already happened, 'anzeigen' should be past tense or omitted altogether. I went for the latter and streamlined all Texts.
"Gesamteinträge", p.e. is meaningless

Reply:
I had a superfluous look at some other language files and corrected some typos. A serious review would require more time though...